### PR TITLE
ci: manual release

### PR DESCRIPTION
**Pull request by kobik**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -56,55 +58,25 @@ jobs:
         run: npm run integration-tests
       - name: coveralls
         uses: coverallsapp/github-action@master
-        if: ${{ matrix.node == '14' }}
+        if: ${{ matrix.NODE_VERSION == '16' }}
```
Comment (ilanKushnir) [inline .github/workflows/main.yml]: shouldn't it be from node 14 and up?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+name: Release
```
Comment (ilanKushnir) [inline .github/workflows/release.yml:1]: what makes this release manual?

Comment (kobik) [reply .github/workflows/release.yml:1]: this
```yaml
on: [ workflow_dispatch ]
```

File: .github/workflows/main.yml (hunk 1 above)
Comment (kobik) [reply .github/workflows/main.yml]: i want to run coveralls only once, so i prefer doing it using the LTS version.

**Pull request merged**